compact/tracking/tof_barrel.xml

```diff
@@ -59,9 +59,11 @@
     <constant name="BarrelTOF_Module_nphi"         value="144"/>
     <constant name="BarrelTOF_Module_nz"           value="1"/>
 
+    <constant name="BarrelTOF_rOffset0"            value="11.6*cm"/>
     <constant name="BarrelTOF_rOffset1"            value="1.6*cm"/>
     <constant name="BarrelTOF_rOffset2"            value="1.4*cm"/>
-    <constant name="BarrelTOF_rmin"                value="BarrelTOF_radius-BarrelTOF_rOffset1"/>
+    <constant name="BarrelTOF_rmin"                value="BarrelTOF_radius-BarrelTOF_rOffset0"/>
+    <constant name="BarrelTOF_rmin1"               value="BarrelTOF_radius-BarrelTOF_rOffset1"/>
```

Comment:
What's the point here? I don't think you need to add these variables with different names, since now `BarrelTOF_rmin` appears unused. This doesn't seem to have changed anything: the rmin used below (rmin1 here) is still the same as before...

Reply:
BarrelTOF_rmin=56 cm is the inner radius of TOF, to be consistent with geometry DB and used in the template variable
BarrelTOF_rmin1=60 cm is the inner radius of TOF currently implemented in DD4HEP, which does not include the support structure at smaller radius